Randolph-dev/Druid_Partnering_Project_React24k
# update database to include missing one missing field

### Related branch:
[SCRUM-155](https://github.com/Randolph-dev/Druid_Partnering_Project_React24k/compare/dev...SCRUM-155-Reformat-About-us-page?expand=1)

### In this PR:
Includes the newest database from Drupal with About Us page content.

### Testing instructions:

- Checkout to this branch.
- Import the latest database.
- Check if it's properly imported to Drupal and whether its content is understandable.

_Note:_ Changes to the page itself to be featured in [SCRUM-155](https://github.com/Randolph-dev/Druid_Partnering_Project_React24k/compare/dev...SCRUM-155-Reformat-About-us-page?expand=1) in the near future.

[SCRUM-155]: https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-155?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[SCRUM-155]: https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-155?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
